File: 724-FindPivotIndex/724-FindPivotIndex.py
# Last updated: 9/14/2025, 9:12:49 AM
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        s,left_sum=sum(nums),0
        for i in range(len(nums)):
            if left_sum==(s-nums[i]-left_sum):
                return i
            left_sum+=nums[i]
        return -1

File: 724-FindPivotIndex/724-FindPivotIndex.java
// Last updated: 9/14/2025, 9:12:52 AM
class Solution {
    public int pivotIndex(int[] nums) {
        int left_sum=0,tot=0;
        for(int x:nums){
            tot+=x;
        }
        for(int i=0;i<nums.length;i++){
            if(left_sum==(tot-nums[i]-left_sum)){
                return i;
            }
            left_sum+=nums[i];
        }
        return -1;
    }
}